docs/io/unified.rst

```diff
@@ -230,6 +230,130 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
+      float64
+      --------
+           1.0
+           2.0
+
+All the other ``mixin columns`` being quite difficult to store
+in FITS tables, due to reasons including extensive metadata and
+no precise mapping to the FITS standard, are treated separately
+by using the argument ``astropy_native`` in the FITS read/write API.
+Thus, an Astropy Table or QTable can read/write these mixin columns
+from/to FITS tables. Currently this is limited to `~astropy.time.Time`
+columns since the FITS standard has been extended to include time.
+
+* Time
+
+  Time as a dimension in astronomical data presents challenges in its 
+  representation in FITS files. The standard has therefore been extended to
+  describe rigorously the time coordinate in the ``World Coordinate System``
+  framework. Refer to `FITS WCS paper IV <http://adsabs.harvard.edu/abs/2015A%26A...574A..36R/>`_
+  for details.
+
+  Allowing `~astropy.time.Time` columns to be written as normal columns in FITS
+  tables thus involves storing time values in a way to ensure retention
+  of precision and mapping the elaborate metadata associated with it to
+  the relevant FITS keywords.
+
+  In accordance with the standard which states that, in binary tables one
+  may use pairs of doubles, the Astropy Time column is written in such a
+  table as a vector of two doubles ``(TFORM n = ‘2D’)`` (jd1, jd2).
+  This reproduces the time values to double-double precision and is the
+  "lossless" version exploiting the higher precision provided in binary
+  tables. Considerable round-tripping of astropy written FITS binary tables
+  containing time columns has been achieved by mapping metadata,
+  ``scale`` and ``singular location`` to corresponding keywords.
+
+  This new functionality has been provided in both, the Table and QTable API,
+  using the optional argument ``astropy_native``. Users can
+  explicitly set it to `True` to allow the writing and reading of Time
+  to and from FITS without much loss of precision and metadata. By default,
+  this functionality is off and users can store the original time
+  representation values in the specified Time ``format`` instead of (jd1, jd2).
+  This might be useful when dealing with other softwares which do not
+  recognize this format or when users want to serialize their Time object
+  in the format of their choosing.
+
+      >>> from astropy.time import Time
+      >>> from astropy.table import Table
+      >>> from astropy.coordinates import EarthLocation
+      >>> t = Table()
+      >>> t['a'] = Time([1,2], scale='tt', format='cxcsec',
+      ...               location=EarthLocation(-2446354, 4237210, 4077985, unit='m'))
+      >>> t.write('my_table.fits', overwrite=True, astropy_native=True)
+      >>> tm = Table.read('my_table.fits', astropy_native=True)
+      >>> tm['a']
+      <Time object: scale='tt' format='jd' value=[ 2450814.50001157  2450814.50002315]>
+      >>> tm['a'].location
+      <EarthLocation (-2446354.,  4237210.,  4077985.) m>
```

Comment:
I would probably chose a format that's allowed in the fits standard such as MJD for simplicity in the first example. Then you can later show this example and say "Hey, look, it's converted to a FITS time automatically and comes correctly when you read it in again.

Also, does tihs example currently work? I thought that's not fully implemented yet?